XenModel/Messages.resx

```diff
@@ -1209,6 +1227,30 @@
   <data name="ACTION_VM_TEMPLATIZING_TITLE" xml:space="preserve">
     <value>Converting VM '{0}' to template</value>
   </data>
+  <data name="ACTION_VUSB_CREATED" xml:space="preserve">
+    <value>Virtual USB created</value>

```

Comment:
I don't think we need to expose the term "virtual USB" to the user. From their point of view, they are attaching an USB device to a VM, so this is what the action should say.

Reply:
Good idea, I'm going to change "virtual USB" into "USB device".